### PR TITLE
TILA-2497 | Application and ApplicationEvent gql additions

### PR DESCRIPTION
**Pull request by nicobav**
## Change log
- Adds possibility to query `Applications` with `appliedCount`
- Adds possibility to query `ApplicationEvents` with `appliedCount`
- Adds possibility to query `ApplicationRounds` with `pk`

### CODE REVIEW

File: api/graphql/application_rounds/application_round_types.py [hunk 1]
```diff
@@ -83,7 +83,6 @@ class Input:
     class Meta:
         model = ApplicationRound
         fields = [
-            "id",
```
Comment (saaste) [inline api/graphql/application_rounds/application_round_types.py:86]: Removing this looks like a breaking change. Hopefully client is not using this field.

File: api/graphql/application_rounds/application_round_types.py [hunk 2]
```diff
@@ -102,12 +101,6 @@ class Meta:
             "target_group",
         ] + get_all_translatable_fields(model)
 
-        filter_fields = {
```
Comment (saaste) [inline api/graphql/application_rounds/application_round_types.py:105]: Same as above. Do we know that client is not using these filters?

**Review by saaste — APPROVED** [review]
Approved with two comments about breaking change. If field/filters are not in use, this is good to go (as soon as tests are green 😄 )! 👍

File: api/graphql/application_rounds/application_round_types.py (hunk 1 above)
Comment (nicobav) [reply api/graphql/application_rounds/application_round_types.py:86]: It shouldn't be in use and I've discussed that these ids are not our way to go, rather the `pk`

File: api/graphql/application_rounds/application_round_types.py (hunk 2 above)
Comment (nicobav) [reply api/graphql/application_rounds/application_round_types.py:105]: Client is using the REST api currently for the application rounds and there are no such filters in use so this should be safe. This was only missing the `pk` filter as was in REST /application_round/<pk>

**Comment by saaste** [comment]
🚢  it! 👏

**Pull request merged**